src/SFA.DAS.AdminService.Web/Controllers/Roatp/Apply/RoatpFinancialController.cs

```diff
@@ -157,13 +157,18 @@ public async Task<IActionResult> GradeApplication(Guid applicationId, [FromForm]
         [HttpPost("/Roatp/Financial/Clarification/{applicationId}")]
         public async Task<IActionResult> SubmitClarification(Guid applicationId, RoatpFinancialClarificationViewModel vm)
         {
```

Comment:
Just a comment to think about... this function is getting chunky and a little difficult to follow, could it be refactored into 2 or 3 smaller private methods?

Reply:
yes, good idea